src/fe_main.ml

```diff
@@ -61,7 +61,7 @@ let _ =
 
   let main_sock = Fecomms.open_unix_domain_sock_server "/var/xapi/forker/main" in
 
-  Stdext.Unixext.pidfile_write !pidfile;
```

Comment:
If we are not writing the pidfile anymore, then all the daemonisation stuff must go...